exercises/E-conditionals/exercise.js

```diff
@@ -9,6 +9,14 @@
 let name = "Daniel";
 let danielsRole = "mentor";
 
+if (danielsRole = "mentor"){
```

Comment:
Comparisons should be ===, in this instance you've used = as assignment and given the variable danielsRole the value "mentor again. 

Be careful as danielsRole = "mentor" will still passes the tests as assignments with strings results to true if the length of the string is greater than 1. Here's an example to show this:
console.log(Boolean((names = "h")));   //true
console.log(Boolean((names = "")));    //false